webui/react/src/styles/antd.scss

```diff
@@ -214,6 +214,10 @@ html {
     border-color: var(--theme-colors-action-normal);
     border-style: solid;
   }
+  .ant-table-thead .ant-table-cell {
+    border-left: 1px solid var(--theme-colors-monochrome-13);
+    border-right: 1px solid var(--theme-colors-monochrome-13);
+  }
```

Comment:
**suggestion:** slightly above this area there are border definitions already defined for header cells, might be better to just tweak the existing ones:

<img width="838" alt="Screen Shot 2021-05-21 at 9 21 07 PM" src="https://user-images.githubusercontent.com/220971/119213174-7d8b7500-ba7a-11eb-8679-11ec8f74e38b.png">